Gemfile

```diff
@@ -7,9 +7,9 @@ gem 'nokogiri'
 
 ### Fastlane Plugins
 
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 6.0'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 6.0'
 #gem 'fastlane-plugin-wpmreleasetoolkit', path: '../../release-toolkit'
-#gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch: '…'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch: 'trunk'
```

Comment:
⚠️ 🚧  Reminder than before merging this PR, we need to revert those temporary changes above and point to the new version of `release-toolkit` after we published it in RubyGems.